client/src/components/Navbar/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Slide from "@material-ui/core/Slide/";
+
+import { Body16R, H18Caps, Body18B } from "../Typograpy";
+import NavLink from "./NavLink";
+import Icon from "../Icon";
+import Image from "../Image";
+
+import {
+  linksTest,
+  memberLinks,
+  earnGroupLinks,
+  staffLinks,
+  spendPartnerLinks,
+} from "./navlinksContnet";
+
+import {
+  Aside,
+  List,
+  Balance,
+  TopNav,
+  MenuBtn,
+  MobileMenu,
+  CloseBtn,
+  MenuContent,
+  BalanceContainer,
+} from "./styled";
+
+const balance = (userType, TCBalance) => (
+  <Balance>
+    <Body16R color="white">
+      {userType === "member" ? "My Balance" : "Balance"}
+    </Body16R>
+    <H18Caps color="white">{TCBalance || "000"} Time Credits</H18Caps>
+  </Balance>
+);
+
+const Navbar = ({ userType, TCBalance }) => {
+  const [open, setOpen] = useState(false);
+  let links = [];
+  switch (userType) {
+    case "member":
+      links = [...memberLinks];
+      break;
+    case "earnGroup":
+      links = [...earnGroupLinks];
+      break;
+    case "spendPartner":
+      links = [...spendPartnerLinks];
+      break;
+    case "staff":
+      links = [...staffLinks];
+      break;
+    default:
+      links = [...linksTest];
+      break;
+  }
+
+  return (
+    <>
+      <Aside>
+        <Image
+          image="logoMed"
+          alt="tempo logo"
+          customStyle={{ padding: "60px 0 0 50px" }}
+        />
+        {userType === "member" && balance(userType, TCBalance)}
```

Comment:
use react functional compoent 
```suggestion
        {userType === "member" && <Balance userType={userType} TCBalance={TCBalance} />}
```

Reply:
done